app/src/main/res/xml/prefs_screen_about.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2014 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+
+
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    android:title="@string/settings_screen_about"
+    android:key="@string/settings_screen_about">
+    <Preference
+        android:icon="@mipmap/ic_launcher"
+        android:title="@string/english_ime_name"
+        android:summary="100% FOSS keyboard, based on AOSP."/>
+    <Preference
+        android:title="@string/version"
+        android:summary="1.4.6"/>
```

Comment:
Can you set the summary in `AboutFragment.java`? Then you could make use of `BuildConfig.VERSION_NAME`.